prelude/cxx/cxx_library.bzl

```
@@ -1295,10 +1374,16 @@ def _form_library_outputs(
             pic: bool,
             optimized: bool,
             debuggable: bool,
+            pch: bool,
             stripped: bool) -> (CxxLibraryOutput | None, LinkInfo | None):
         if not compile_output:
             return (None, None)
-        objects = compile_output.stripped_objects if stripped else compile_output.objects
+
+        if pch:
+            objects = compile_output.pch_object_output
+        else:
+            objects = compile_output.stripped_objects if stripped else compile_output.objects
+
```

Can we just avoid Yet Another Bool Arg here?
```suggestion
        objects = compile_output.pch_object_output if compile_output.pch_object_output else compile_output.stripped_objects if stripped else compile_output.objects
```